name.abuchen.portfolio.ui/src/name/abuchen/portfolio/ui/messages.properties

```diff
@@ -1694,6 +1694,10 @@ MsgInfoChangingCurrencyNotPossible = This security has already transactions reco
 
 MsgInfoRetiredSecurities = Inactive securities are hidden in transactions dialogs (e.g. buy and sell dialogs).\nExisting transactions are not touched.\nQuotes are still updated automatically unless the security is configured as "no automatic download".
 
+MsgInvalidImage = Invalid image file
+
+MsgInvalidImageDetail = Provided file ''{0}'' does not appear to be an image or could not be read. 
```

Comment:
Wenn es möglich ist, könnten gleich die Übersetzungen für die jeweilgen anderen Sprachen eingefügt werden:
French: `\u00c0 condition que le fichier ''{0}'' ne semble pas \u00eatre une image ou ne puisse pas \u00eatre lu.`
Portugais: `O ficheiro fornecido''{0}'' n\u00e3o pare\u00e7a ser uma imagem ou n\u00e3o  possa ser lido.`
Neerlandês: `Het bestand ''{0}'' lijkt geen afbeelding te zijn of kon niet worden gelezen.`
Espanhol: `El archivo proporcionado''{0}'' no parece ser una imagen o no se puede leer.`